app/models/podcast.rb

```diff
@@ -26,7 +26,7 @@ class Podcast < ApplicationRecord
   # these keep changing - so just translate to the current accepted values
   VALID_EXPLICITS = %w[true false]
   EXPLICIT_ALIASES = {
-    "" => "false",
+    "" => nil,
```

Comment:
This seemed like an oversight in Publish ... you should be able to set this to `nil` on Episodes.